packages/fxa-settings/src/components/Settings/FlowRecoveryKeyDownload/en.ftl

```diff
@@ -2,14 +2,8 @@
 ## Users see this view when they are generating a new account recovery key
 ## This screen displays the generated key and allows users to download or copy the key
 
-flow-recovery-key-download-heading = Account recovery key generated — store it in a place you’ll remember
+flow-recovery-key-download-heading-v2 = Account recovery key created— Download and store it now
 # The "key" here refers to the term "account recovery key"
-flow-recovery-key-download-info = This key will help recover your data if you forget your password.
-# This heading is shown above a list of options for storing the account recovery key
-flow-recovery-key-download-storage-ideas-heading = Some ideas for storing your account recovery key:
-flow-recovery-key-download-storage-ideas-folder = Memorable folder in your device
-flow-recovery-key-download-storage-ideas-cloud = Trusted cloud storage
-flow-recovery-key-download-storage-ideas-print = Print and keep a physical copy
-flow-recovery-key-download-storage-ideas-pwd-manager = Password manager
+flow-recovery-key-download-info-v2 = This key allows you recover your data if you forget your password. Download it now and keep it somewhere you’ll remember— you won’t be able to return to this page later.
```

Comment:
Related to @bcolsson's suggestion above (re: space after `—`):
```suggestion
flow-recovery-key-download-info-v2 = This key allows you recover your data if you forget your password. Download it now and keep it somewhere you’ll remember — you won’t be able to return to this page later.
```

Reply:
@pdehaan - Nice catch, +1 to this.